Mathlib/NumberTheory/LSeries/Basic.lean

```diff
@@ -186,6 +192,21 @@ theorem LSeriesSummable_iff_of_re_eq_re {f : ℕ → ℂ} {s s' : ℂ} (h : s.re
 #align nat.arithmetic_function.l_series_summable_iff_of_re_eq_re LSeriesSummable_iff_of_re_eq_re
 
 
+/-!
+### Notation
+-/
+
+@[inherit_doc]
+scoped[LSeries.notation] notation "L" => LSeries
```

Comment:
Why have a separate `LSeries.notation` namespace? If you put it in the namespace `LSeries` then you can still have the effect of importing the notation, but not the theorem names, by doing `open scoped LSeries`.

Reply:
I want to leave the option of importing the theorem names, but not the notation.
(I noticed that opening the `ArithmeticFunction` namespace also imports, e.g., the notation `ζ` for a specific arithmetic function, and in some cases I'd like to use the theorems without prefix, but not necessarily the notation.)